amd_openvx_extensions/amd_opencv/include/vx_ext_opencv.h

```diff
@@ -83,6 +98,10 @@ extern "C"
 	 * \param [in] ddepth The input <tt>\ref VX_TYPE_INT32</tt> scalar to set output image depth.
 	 * \param [in] kwidth The input <tt>\ref VX_TYPE_UINT32</tt> scalar to set box filter width.
 	 * \param [in] kheight The input <tt>\ref VX_TYPE_UINT32</tt> scalar to set box filter height.
+	 * \param [in] Anchor_X The input <tt>\ref VX_TYPE_UINT32</tt> scalar to set anchor point x.
+	 * \param [in] Anchor_Y The input <tt>\ref VX_TYPE_UINT32</tt> scalar to set anchor point y.
+	 * \param [in] Normalized The input <tt>\ref VX_TYPE_BOOL</tt> scalar for Normalized argument.
+	 * \param [in] Bordertype The input <tt>\ref VX_TYPE_UINT32</tt> scalar to set borderType.
```

Comment:
Should the args be in CAPS ? The rest of the arguments seem to be all in small letters. Please check and confirm on this